presto-native-execution/presto_cpp/main/PrestoTask.cpp

```diff
@@ -117,16 +118,27 @@ static void addRuntimeMetric(
   }
 }
 
-// Updates a Velox runtime metric in the unordered map if the value is not 0.
-static void addRuntimeMetricIfNotZero(
-    std::unordered_map<std::string, RuntimeMetric>& runtimeMetrics,
+static void setProtocolRuntimeStats(
+    protocol::RuntimeStats& runtimeStats,
     const std::string& name,
-    uint64_t value) {
-  if (value > 0) {
-    auto veloxMetric =
-        createVeloxRuntimeMetric(value, RuntimeCounter::Unit::kNone);
-    addRuntimeMetric(runtimeMetrics, name, veloxMetric);
+    const RuntimeMetric& metric,
+    bool requirePositive) {
+  if (requirePositive && metric.sum == 0) {
```

Comment:
This line is confusing to read. `requirePositive` means it needs to be greater than zero. A better name can be `includeZeroValueMetric`. Also good to have documentation what this parameter is doing and why we skip it sometimes.

Reply:
Sure, either way for the config works for me